[
   {
      "author_association" : "MEMBER",
      "body" : "Not sure what is going on here. I assume it is a fault on the ci provider's side. I can try switching the virtualization to another one to see if it helps. If not, we might have to run the ci on our own hardware.",
      "created_at" : "2020-10-08T12:52:59Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/20093#issuecomment-705547164",
      "id" : 705547164,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/20093",
      "node_id" : "MDEyOklzc3VlQ29tbWVudDcwNTU0NzE2NA==",
      "performed_via_github_app" : null,
      "updated_at" : "2020-10-08T12:52:59Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/705547164",
      "user" : {
         "avatar_url" : "https://avatars0.githubusercontent.com/u/6399679?v=4",
         "events_url" : "https://api.github.com/users/MarcoFalke/events{/privacy}",
         "followers_url" : "https://api.github.com/users/MarcoFalke/followers",
         "following_url" : "https://api.github.com/users/MarcoFalke/following{/other_user}",
         "gists_url" : "https://api.github.com/users/MarcoFalke/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/MarcoFalke",
         "id" : 6399679,
         "login" : "MarcoFalke",
         "node_id" : "MDQ6VXNlcjYzOTk2Nzk=",
         "organizations_url" : "https://api.github.com/users/MarcoFalke/orgs",
         "received_events_url" : "https://api.github.com/users/MarcoFalke/received_events",
         "repos_url" : "https://api.github.com/users/MarcoFalke/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/MarcoFalke/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/MarcoFalke/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/MarcoFalke"
      }
   }
]
